src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md

```diff
@@ -20,11 +20,11 @@
 ## Current Release
 
 ## Version 5.0.5
-* Fixed issue with default resource groups not being set.
-* Updated common runtime assemblies
+* Added Standard_Microsoft in Cdn pricing sku.
```

Comment:
move this to under Current Release

And restore the values under the old version numbers as they were